docs/code-usage.rst

```diff
@@ -0,0 +1,178 @@
+.. index::
+   single: Phinx via Code
+
+Calling Phinx via Code
+===================================
+
+Phinx is primarily a command line tool, however Phinx does support code-based calls.
+
+.. warning::
+
+    Not all terminal commands are available to be used within code yet.
+
+Initial Setup
+--------------
+
+.. note::
+
+    For Phinx via Code to work at all, you need to ensure that Phinx is accessible via autoloading,
+    the best way to manage this would be to ``require "vendor/autoload.php";`` the composer autoloader.
+
+Phinx will require instantiating, which can be achieved by the following.
+
+.. code-block:: php
+
+    $app = new \Phinx\Console\PhinxApplication($version);
+    $phinx = new \Phinx\Wrapper\TextWrapper($app);
+
+Where ``$version`` is the version number of Phinx (eg. '0.4.1').

```

Comment:
I have to disagree.

Many people will already have composer's autoloading defined, and thus requiring phinx.php will autoload PhinxApplication twice.

Also, I think this:
    $app= new PhinxApplication();
is clearer than
    $app= require 'vendor/robmorgan/phinx/app/phinx.php';
because its immediately clear whats going on.